app/views/forms/edit/edit-resource-schema-base-directive.js

```diff
@@ -275,6 +285,21 @@ define(['app', 'lodash','text!views/forms/edit/edit-resource-schema-base-directi
 					}
 				}
 
+				$scope.onRaRecommendChange = function(value){
+					if(!value){
+						$scope.document.bchRaAuthorAffiliation = undefined;
+						$scope.document.bchRaSubjects = undefined;
+					}
+				}
+
+				$scope.onThematicAreaChange = function(value){
+					if (value && _.indexOf((_.map(value, "identifier")), 'FBAF958B-14BF-45DD-BC6D-D34A9953BCEF') >= 0){
+						$scope.displayBchRaSubjects = true;
+					}
+					else
+						$scope.displayBchRaSubjects = false;
```

Comment:
```suggestion
$scope.displayBchRaSubjects = _.some(value||[], {identifier: "FBAF958B-14BF-45DD-BC6D-D34A9953BCEF"};
```

please use `hasRiskAssessmentSubject`

Reply:
used hasRiskAssessmentSubject